test/safe.test.js

```diff
@@ -1,377 +1,190 @@
+import { SAFE_LAST_VERSION, SAFE_CRC_VERSION } from '~/common/constants';
+import { SafeDeployedError } from '~/common/error';
+
 import createCore from './helpers/core';
-import getAccount from './helpers/account';
+import accounts from './helpers/accounts';
 import web3 from './helpers/web3';
-import isContractDeployed from './helpers/isContractDeployed';
-import {
-  addTrustConnection,
-  deploySafeAndToken,
-  fundSafe,
-  deployCRCVersionSafe,
-  deployToken,
-} from './helpers/transactions';
-
-import { SAFE_LAST_VERSION, SAFE_CRC_VERSION } from '~/common/constants';
+import { deployCRCVersionSafe } from './helpers/transactions';
+import generateSaltNonce from './helpers/generateSaltNonce';
 
 describe('Safe', () => {
-  let core;
-  let accounts;
-
-  beforeAll(() => {
-    accounts = new Array(4).fill({}).map((item, index) => {
-      return getAccount(index);
-    });
-
-    core = createCore();
-  });
+  const core = createCore();
 
   describe('when a new Safe gets manually created', () => {
+    const nonce = generateSaltNonce();
     let safeAddress;
-    let nonce;
 
-    beforeAll(async () => {
-      nonce = Date.now();
-      safeAddress = await core.safe.prepareDeploy(accounts[0], {
-        nonce,
-      });
-    });
+    it('should deploy a Safe successfully', async () => {
+      safeAddress = await core.safe.deploySafe(accounts[0], { nonce });
 
-    it('should be a valid address', () => {
       expect(web3.utils.isAddress(safeAddress)).toBe(true);
-    });
-
-    it('should have predicted its future Safe address', async () => {
-      const predictedSafeAddress = await core.safe.predictAddress(accounts[0], {
-        nonce,
-      });
 
-      expect(web3.utils.isAddress(predictedSafeAddress)).toBe(true);
-      expect(predictedSafeAddress).toBe(safeAddress);
+      return core.safe
+        .isDeployed(accounts[0], { safeAddress })
+        .then((isDeployed) => expect(isDeployed).toBe(true));
     });
 
-    it('should return the correct status', async () => {
-      const status = await core.safe.getSafeStatus(accounts[0], {
-        safeAddress,
-      });
-
-      expect(status.isCreated).toBe(true);
-      expect(status.isDeployed).toBe(false);
-    });
-
-    it('should recover the same Safe address when doing it again', async () => {
-      const safeAddressAgain = await core.safe.prepareDeploy(accounts[0], {
-        nonce,
-      });
-
-      expect(safeAddressAgain).toBe(safeAddress);
-    });
-
-    it('should be able to manually fund it for deployment', async () => {
```

Comment:
this is also missing right - we said that we will enable this as it is now.

Reply:
It is still there to remind that change in the futuro and do not forget.